app/settings.py

```diff
@@ -20,13 +24,16 @@
 if SDX_RECEIPT_RRM_SECRET is not None:
     SDX_RECEIPT_RRM_SECRET = SDX_RECEIPT_RRM_SECRET.encode("ascii")
 
+RM_SDX_GATEWAY_URL = os.getenv("RM_SDX_GATEWAY_URL", "http://localhost:8191/receipts")
+
```

Comment:
I was just wondering why it was 8191 and then it occurred to me we should update the sdx-mock-receipt service to cope with case id and then default to the port of that service. Otherwise this won't work in sdx-compose.